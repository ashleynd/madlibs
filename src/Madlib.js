import React from "react";

const Madlib = ({id, noun, noun2, adjective, color}) => {
    return (
        <div>
        <h2>The {adjective} {noun} jumped over the {color} {noun2}.</h2>
        {/* <img className="gif" alt="gif" src="https://media.giphy.com/media/fnK0jeA8vIh2QLq3IZ/giphy.gif"></img> */}
        </div>
    )
}

export default Madlib;

