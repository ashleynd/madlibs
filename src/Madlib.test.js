import React from "react";
import { render } from "@testing-library/react";
import Madlib from "./Madlib";

it("renders without crashing", function() {
  render(<Madlib />);
});

it("matches snapshot", function() {
  const { asFragment } = render(<Madlib />);
  expect(asFragment()).toMatchSnapshot();
});