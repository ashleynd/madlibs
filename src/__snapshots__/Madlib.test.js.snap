// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <h2>
    The   jumped over the  .
  </h2>
  <img
    alt="gif"
    class="gif"
    src="https://media.giphy.com/media/fnK0jeA8vIh2QLq3IZ/giphy.gif"
  />
</DocumentFragment>
`;
